DirectProgramming/DPC++FPGA/Tutorials/Features/mem_channels/README.md

```diff
@@ -0,0 +1,306 @@
+   Alternatively, to compile for the Intel® FPGA PAC D5005 (with Intel Stratix®
+   10 SX), run `cmake` using the command:
+
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
+   ```
+   You can also compile for a custom FPGA platform. Ensure that the board
+   support package is installed on your system. Then run `cmake` using the
+   command:
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=<board-support-package>:<board-variant>
+   ```
+
+2. Compile the design through the generated `Makefile`. The following build
+   targets are provided, matching the recommended development flow:
+
+   * Compile for emulation (fast compile time, targets emulated FPGA device): 
+     ```
+     nmake fpga_emu
+     ```
+   * Generate the optimization report: 
+     ```
+     nmake report
+     ``` 
+   * Compile for FPGA hardware (longer compile time, targets FPGA device):
+     ```
+     nmake fpga
+     ``` 
+
+*Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005
+(with Intel Stratix® 10 SX) do not support Windows*. Compiling to FPGA hardware
+on Windows* requires a third-party or custom Board Support Package (BSP) with
+Windows* support.
+ 
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the
+Visual Studio* IDE (in Windows*). For instructions, refer to the following
+link: [Intel® oneAPI DPC++ FPGA Workflows on Third-Party
+IDEs](https://software.intel.com/en-us/articles/intel-oneapi-dpcpp-fpga-workflow-on-ide)
+
+
+## Examining the Reports
+Locate the pair of `report.html` files in the `mem_channels_interleaving.prj`
+and `mem_channels_no_interleaving.prj` directories. Open the reports in any of
+Chrome*, Firefox*, Edge*, or Internet Explorer*. In the "Summary" tab, locate
+the "Quartus Fitter Resource Utilization Summary" entry and expand it to see
+the table showing the FPGA resources that were allocated for the design. Notice
+that when burst-interleaving is disabled, the FPGA resources required are
+significantly lower than the case where burst-interleaving is enabled.
+
+
+## Running the Sample
+
+ 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+     ```
+     ./mem_channels.fpga_emu     (Linux)
+     mem_channels.fpga_emu.exe   (Windows)
+     ```
+    Note that the `mem_channel` property and the `-Xsno-interleaving` flag have
+    no impact on the emulator which is why we only have a single executable for
+    this flow.
+2. Run the sample on the FPGA device (two executables should be generated):
+     ```
+     ./mem_channels_interleaving.fpga         (Linux)
+     ./mem_channels_no_interleaving.fpga         (Linux)
+     ```
+
+### Example of Output
+
+Running `./mem_channels_interleaving.fpga`:
+```
+Vector size: 100000 
+Verification PASSED
+
+Kernel execution time: 0.004004 seconds
+Kernel throughput 749.230914 MB/s
+```
+
+Running `./mem_channels_interleaving.fpga`:
```

Comment:
Can we name the two executables something different to avoid confusion which one is with interleaving and which one is without.

Reply:
Addressed. This was a typo actually - thanks for catching it.